### PR TITLE
Update ARK RTK GPS docs

### PR DESCRIPTION
**Pull request by AlexKlimaj**

### CODE REVIEW

File: en/dronecan/ark_rtk_gps.md [hunk 1]
```diff
@@ -141,14 +121,51 @@ Pin | Name
...
+  - Slow blinking green and blue is CAN enumeration
+  - Slow blinking green, blue, and red is firmware update in progress
+  - Blinking red is error
+    - If you see a red LED there is an error and you should check the following
+      - Make sure the flight controller has an SD card installed
+      - Make sure the ARK RTK GPS has `ark_can-rtk-gps_canbootloader` installed prior to flashing `ark_can-rtk-gps_default`
+      - Remove binaries from the root and ufw directories of the SD card and try to build and flash again
 
-If you see a red LED there is an error and you should check the following:
+### Updating Ublox F9P Module
```
Comment (hamishwillee) [inline en/dronecan/ark_rtk_gps.md:148]: Yes, moving this here is much better.

File: en/gps_compass/u-blox_f9p_heading.md [hunk 2]
```diff
@@ -29,14 +29,7 @@ Ideally the two antennas should be identical, on the same level/horizontal plane
...
 
-- The CAN port of the GPS devices need to be connected together and one connected to the autopilot.
-- Choose one GPS module to be the *Rover* and the other to be the *Moving Base*.
-  - On the *Rover*, set [GPS_UBX_MODE](../advanced_config/parameter_reference.md#GPS_UBX_MODE) to `1`.
-  - On the *Moving Base*, set [GPS_UBX_MODE](../advanced_config/parameter_reference.md#GPS_UBX_MODE) to `2`.
-- [EKF2_GPS_CTRL](../advanced_config/parameter_reference.md#EKF2_GPS_CTRL) parameter bit 7 must be set (see [GPS > Configuration > GPS as Yaw/Heading Source](../gps_compass/README.md#configuring-gps-as-yaw-heading-source)).
-- [GPS_YAW_OFFSET](../advanced_config/parameter_reference.md#GPS_YAW_OFFSET) may need to be set (see [GPS > Configuration > GPS as Yaw/Heading Source](../gps_compass/README.md#configuring-gps-as-yaw-heading-source)).
-- Reboot and wait until both devices have GPS reception.
-  `gps status` should then show the *Rover* GPS going into RTK mode, which means the heading angle is available.
+- See [ARK RTK GPS](../dronecan/ark_rtk_gps.md#setting-up-moving-baseline-gps-heading)
```
Comment (hamishwillee) [inline en/gps_compass/u-blox_f9p_heading.md:32]: @AlexKlimaj This is the top level document on CAN setup - the theory is that it is generic and other system link to it and share the information. Other vendors that support this feature are not going to want to on-link to ARK RTK documents.
It is OK to have duplicate material in ARK RTK GPS.

Can you revert this and make sure that it is up to date?

Comment (AlexKlimaj) [reply en/gps_compass/u-blox_f9p_heading.md]: At the moment, this setup is specific to the ARK RTK GPS. I think we just want this generic and to refer to the CAN RTK GPS docs to instructions.

Comment (hamishwillee) [reply en/gps_compass/u-blox_f9p_heading.md]: Thanks. I like your fix on this.

**Comment by hamishwillee** [comment]
THanks very much. merging.

**Pull request merged**